src/apps/intel/intel10g.lua

```diff
@@ -187,6 +192,21 @@ do
    end
 end
 
+function M_sf:poll_ingress_packet_drops ()
+   self.qprdc = 0
+   local function fn ()
+      self.qprdc = tonumber(self.qs.QPRDC[0]())
+   end
+   local interval = 1e9 * ingress_packet_drops.poll_timer
+   local t = timer.new("ingress_packet_drops", fn, interval, 'repeating')
+   timer.activate(t)
+   return self
+end
+
+function M_sf:ingress_packet_drops ()
+   return self.qprdc
```

Comment:
also does reading the qprdc reset it to zero or something?  otherwise not sure how SF configurations would reset.  (edit: never mind, apparently the comment below indicates that reading it resets the value)